test-sql/numeric.sql

```diff
@@ -0,0 +1,11 @@
+SELECT omt_as_numeric(null) AS fail;
```

Comment:
I think it would be better to give each alias a unique name - this way it is easier to match the output file with each query.  E.g. here it could be `AS fail_null`

Reply:
yep, I initially did that in fact, agree that would help when errors raise.